en/advanced/gimbal_control.md

```diff
@@ -1,82 +1,64 @@
 
-PX4 contains a generic mount/gimbal control driver with different input and output methods.
-- The input defines how you control the gimbal: via RC or via MAVLink commands (for example in missions or surveys).
-- The output defines how the gimbal is connected: either via MAVLink commands or using the Flight Controller AUX PWM port.
-Any input method can be selected to drive any output, and both input and output have to be configured via parameters.
+## Overview
 
-## Parameters
+PX4 contains a generic mount/gimbal control driver that supports different input and output methods:
 
-The [Mount](../advanced_config/parameter_reference.md#mount) parameters are used to setup the mount driver.
+- The input method defines the protocol used to command a gimbal mount that is managed by PX4.
+  This might be an RC controller, a MAVLink command sent by a GCS, or both — automatically switching between them.
+- The output method defines how PX4 communicates with the connected gimbal.
+  The recommended protocol is MAVLink v2, but you can also connect directly to a Flight Controller PWM output port.
 
-The most important ones are the input ([MNT_MODE_IN](../advanced_config/parameter_reference.md#MNT_MODE_IN)) and the output ([MNT_MODE_OUT](../advanced_config/parameter_reference.md#MNT_MODE_OUT)) mode.
-By default, the input is disabled and the driver does not run.
-After selecting the input mode, reboot the vehicle so that the mount driver starts.
+PX4 takes the input signal and routes/translates it to be sent through to the output (any input method can be selected to drive any output).
 
-If the input mode is set to `AUTO`, the mode will automatically be switched based on the latest input.
-To switch from MAVLink to RC, a large stick motion is required.
+Both the input and output are configured using parameters.
+The input is set using the parameter [MNT_MODE_IN](../advanced_config/parameter_reference.md#MNT_MODE_IN).
+By default this is set to `Disabled (-1)` and the driver does not run.
+After selecting the input mode, reboot the vehicle to start the mount driver.
+
+You should set `MNT_MODE_IN` to one of: `RC (1)`, `MAVlink gimbal protocol v2 (4)` or `Auto (0)` (the other options are deprecated).
+If you select `Auto (0)`, the gimbal will automatically select either RC or or MAVLink input based on the latest input (note that to switch from MAVLink to RC requires a large stick motion).
+
+The output is set using the [MNT_MODE_OUT](../advanced_config/parameter_reference.md#MNT_MODE_OUT) parameter.
+This is set to output on a PXM port by default (`AUX (0)`).
+If the [MAVLink Gimbal Protocol v2](https://mavlink.io/en/services/gimbal_v2.html) is supported by your gimbal, you should instead use/select `MAVLink gimbal protocol v2 (2)`.
+
+The full list of parameters for setting up the mount driver can be found in [Parameter Reference > Mount](../advanced_config/parameter_reference.md#mount) (the most relevant values are described below).
```

Comment:
There are a whole bunch of offset parameters etc - I assume these are all still "valid" in the age of control allocation?

Reply:
Yes.